pyne/ionization_calculator.py

```diff
@@ -0,0 +1,598 @@
+
+
+
+Fm = (5.551, 12.14, 24.36, 48.58, 72.80, 88.96, 105.0, 120.9, 136.9, 161.0, 185.2, 209.4, 236.5, 256.8, 276.2, 300.4, 325.7, 351.1, 376.4, 401.7, 443.5, 464.9, 606.4, 642.2, 678.0, 713.8, 749.5, 785.3, 838.1, 875.2, 912.3, 949.4, 1110, 1151, 1192, 1233, 1372, 1418, 1548, 1596, 1767, 1855, 1944, 2033, 2121, 2299, 2388, 2499, 2586, 2766, 2856, 2947, 3395, 3481, 3566, 3652, 3737, 3823, 4019, 4107, 4196, 4284, 4655, 4742, 4829, 4916, 5427, 5526, 5822, 5920, 8574, 8739, 9003, 9217, 9431, 10120, 10350, 10570, 10800, 11430, 11620, 11810, 12010, 13840, 14070, 14620, 14830, 28870, 29400, 29940, 30470, 36860, 37540, 38900, 39450, 155500, 157800)
+
+
+
+Md = (5.624, 12.29, 25.33, 50.32, 73.96, 90.18, 106.4, 122.6, 140.2, 165.1, 190.1, 215.1, 240.1, 261.8, 281.5, 307.2, 333.3, 359.4, 385.4, 411.5, 437.6, 474.3, 496.1, 649.2, 685.7, 722.1, 758.6, 795.0, 831.4, 884.9, 922.7, 960.6, 998.4, 1164, 1205, 1247, 1288, 1435, 1482, 1615, 1664, 1843, 1934, 2024, 2294, 2385, 2475, 2585, 2677, 2769, 2861, 2952, 3044, 3498, 3585, 3672, 3759, 3845, 3932, 4134, 4224, 4314, 4404, 4780, 4868, 4956, 5044, 5585, 5685, 5987, 6087, 8796, 9013, 9230, 9447, 9663, 9880, 10370, 10600, 10830, 11060, 11700, 11890, 12090, 12280, 14200, 14430, 15000, 15210, 29510, 30050, 30590, 31130, 37830, 38530, 39910, 40470, 159400, 161700)
+
+
+
+No = (5.697, 12.45, 26.27, 52.00, 74.92, 91.35, 107.8, 124.2, 143.4, 169.2, 194.9, 220.6, 246.3, 266.8, 286.9, 312.2, 340.6, 367.4, 394.2, 421.0, 447.8, 474.7, 506.1, 528.3, 693.3, 730.4, 767.5, 804.6, 841.7, 878.8, 932.8, 971.4, 1010, 1049, 1218, 1261, 1303, 1345, 1499, 1547, 1684, 1733, 1921, 2013, 2105, 2197, 2288, 2380, 2472, 2564, 2676, 2770, 2863, 2957, 3050, 3143, 3603, 3691, 3779, 3867, 3955, 4043, 4252, 4343, 4434, 4525, 4907, 4996, 5085, 5175, 5746, 5848, 6156, 6257, 9021, 9241, 9460, 9679, 9898, 10120, 10630, 10960, 11320, 11970, 12170, 12370, 12560, 14570, 14800, 15390, 15600, 30160, 30700, 31250, 31800, 38800, 39550, 40940, 41510, 163300, 165700)
+
+
+
+Lw = (4.312, 14.00, 21.36, 42.78, 70.16, 90.13, 107.2, 124.3, 141.3, 165.8, 193.2, 220.5, 247.9, 269.4, 290.1, 316.7, 344.1, 374.5, 402.7, 430.9, 459.2, 487.4, 513.6, 536.5, 561.3, 742.2, 780.0, 817.9, 855.7, 893.6, 931.4, 987.4, 1027, 1066, 1105, 1280, 1322, 1322, 1365, 1408, 1570, 1619, 1760, 1810, 2007, 2100, 2193, 2287, 2380, 2473, 2566, 2660, 2775, 2870, 2965, 3060, 3155, 3250, 3715, 3804, 3894, 3983, 4072, 4161, 4377, 4470, 4562, 4655, 5042, 5132, 5915, 6019, 6333, 6436, 9255, 9477, 9698, 9920, 10400, 10360, 10890, 11120, 11360, 11590, 12250, 12850, 14950, 15190, 15790, 16010, 30820, 31370, 31920, 32470, 39860, 40590, 42010, 42590, 167300, 169700)
+
+
+def function(element,charge):
+
+return elements[index][charge]
```

Comment:
you want to indent this line if your desire is to have function(element, charge) call the return line. 

Also, index should not be defined globally as it currently is defined in line 12. It should be defined within this function if you want it to be part of this function.

Finally, give "function" a meaningful name... 

```
def get_ionization_potential(element, charge):
    index = nucname.znum(element)
    return elements[index][charge]
```